### ISSUE TITLE
selector-no-redundant-nesting-selector Doesn't Error On Multi-part Selectors

### ISSUE THREAD

**Issue by epmatsw** [kristerkari/stylelint-scss#112]
```
.a {
    & .b,
    & .c {

    }
}
```

An error is correctly reported on the `& .b` line, but no error is reported on the `& .c` line. Seems like all of the pieces of the selector need to be examined by this rule.
